.changeset/curly-fireants-jog.md

```diff
@@ -0,0 +1,10 @@
+---
+"create-eth": patch
+---
+
+- remove chainId from useContractWrite (#584)
+- Add copy to clipboard to TxReceipt (#590)
+- Add event logs to transaction page (#591)
+- deployedContracts & externalContracts (#592)
+- Fix typos in comments (#596)
+- Show inherited functions in Debug (when deploying with hardhat) (#564)
```

Comment:
```suggestion
- Fix typos in comments (#596)
- Show inherited functions in Debug (when deploying with hardhat) (#564)
```

We should only keep last two since we already added others in last changeset at #595